docs/source/resources/package-spec.rst

```diff
@@ -314,7 +314,7 @@ parts:
 Remember that the version specification cannot contain spaces,
 as spaces are used to delimit the package, version, and build
 string in the whole match specification. ``python >= 2.7`` is an
-invalid match specification. Furthermore, ``python>=2.7`` is
+invalid match specification. Furthermore,"``python >= 2.7``" with double quotes is
```

Comment:
I have reviewed the document again.
Kindly review my changes again.

[https://github.com/conda/conda-build/pull/4782#discussion_r1187590233](https://github.com/conda/conda-build/pull/4782#discussion_r1187590233)